.github/workflows/rust.yml

```diff
@@ -70,3 +70,7 @@ jobs:
     - name: Benchmark
       if: matrix.rust == 'nightly'
       run: cargo +${{ matrix.rust }} bench
+
+    - name: Minimal versions
+      if: matrix.rust == 'nightly'
+      run: cargo +${{ matrix.rust }} -Zdirect-minimal-versions build
```

Comment:
I personally think we might as well do cargo test rather than just cargo build

Reply:
So updated!